### ISSUE TITLE
hyperlink add-ons

### ISSUE THREAD

**Issue by m9ac** [Manuel83/craftbeerpi3#73]
If the URL listed along with each add-on were hyperlinked to its corresponding git page, that would be an improvement most could appreciate. This project is really looking amazing!

**Comment by JamFfm** [comment]
Yes it would be helpful to hyperlink the git-url of the plugins in the plugin info window
And Project is amazing!!

**Comment by JamFfm** [comment]
pleaase close, this one is fixed
